fvm/environment/meter.go

```diff
@@ -35,8 +35,8 @@ const (
 	ComputationKindRemoveAccountContractCode
 	ComputationKindResolveLocation
 	ComputationKindRevokeAccountKey
-	ComputationKindRevokeEncodedAccountKey
-	_
```

Comment:
Is Encoded Account keys not something supported anymore?

Reply:
The functions to add account keys via "encoded keys" has been deprecated when the object-oriented key management functions got added, a couple years ago. The deprecated functions  got finally removed.